engines/bastion_katello/app/assets/javascripts/bastion_katello/products/details/repositories/details/views/repository-info.html

```diff
@@ -20,6 +20,13 @@ <h4 translate>Basic Information</h4>
       <dt translate>Type</dt>
       <dd>{{ repository.content_type }}</dd>
 
+      <dt translate>Arches</dt>
```

Comment:
I would maybe re-word this to something to indicate that this is restricting client usage, maybe something like "Restrict to clients of Arch:"